src/main/webapp/app/modeling-editor/modeling-editor.component.ts

```diff
@@ -165,6 +140,7 @@ export class ModelingEditorComponent implements AfterViewInit, OnDestroy, OnChan
             // Check if the Person class is correct
             case personUML.name: {
                 const nameAttribute = this.elementWithAttribute(personUML.attribute, umlModel);
+                // const emptyMethod = this.elementWithMethod('', umlModel);
```

Comment:
do we need this comment or can we get rid of it?

Reply:
Yes, I will delete it.